services/movie.py

```diff
@@ -23,18 +28,20 @@ def get_movie_by_id(movie_id: int) -> Movie:
 
 
 def create_movie(
-    movie_title: str,
-    movie_description: str,
-    genres_ids: list = None,
-    actors_ids: list = None,
+        movie_title: str,
+        movie_description: str,
+        genres_ids: list = None,
+        actors_ids: list = None,
```

Comment:
same here and farther. there should be 4 spaces not 8

Reply:
So that there are no differences in the code

Reply:
This is code style. Indentation in Python is 4 spaces. We can use 8 in function declaration to distinguish parameters. But here it's not necessary because they will be distinguished with 4 spaces as well